test/config/TestConfig.cpp

```diff
@@ -14,14 +14,22 @@ using namespace Aws::Crt;
 using namespace Aws::Iot::DeviceClient;
 using namespace Aws::Iot::DeviceClient::Util;
 
+TEST(Config, CreateTempFile)
+{
+    string filePath = "/tmp/test";
+    ofstream file(filePath, std::fstream::app);
+    file << "test message" << endl;
+    ASSERT_TRUE(FileUtils::FileExists(filePath));
+}
+
 TEST(Config, AllFeaturesEnabled)
 {
     constexpr char jsonString[] = R"(
 {
     "endpoint": "endpoint value",
-    "cert": "cert",
-    "key": "key",
-    "root-ca": "root-ca",
+    "cert": "/tmp/test",
```

Comment:
update this to "/tmp/aws-iot-device-client-test-file" since it will be created in users linux machine, it is better to name it after device client.